docs/using/frameworks/index.rst

```diff
@@ -20,6 +20,15 @@ streamline the process of adding oso to your application.
     def secret_route():
         return "Hello world"
 
+Python
```

Comment:
We should remove the "Coming soon" section above

Reply:
And I would just link the Flask page instead of doing a toctree, so it just shows up as `Flask`